File: HackerRank/30-Days-of-Code/C++/day16-StringToInteger.cpp
#include <iostream>
#include <string>
#include <exception>
using namespace std;

// Question Link : https://www.hackerrank.com/challenges/30-exceptions-string-to-integer/problem

int main()
{
    string S;
    getline(cin, S);
    
    try{
        int stringToInt =stoi(S);
        cout << stringToInt;
    }
    catch(exception e){
        cout << "Bad String";
    }

    return 0;
}

File: HackerRank/30-Days-of-Code/Java/day16StringtoInteger.java
import java.util.Scanner;

// Question Link : https://www.hackerrank.com/challenges/30-exceptions-string-to-integer/problem

public class day16StringtoInteger {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        String S = scan.nextLine();
        
        try{
            int stringtoInteger = Integer.parseInt(S);
            System.out.println(stringtoInteger);
        }
        catch(Exception e){
            System.out.println("Bad String");
        }
    }
}
